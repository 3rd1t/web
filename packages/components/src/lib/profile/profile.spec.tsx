import React from "react"
import { render } from "@testing-library/react"

import Profile from "./profile"

describe("Profile", () => {
  it("should render successfully", () => {
    const { baseElement } = render(
      <Profile className="w-1/4" name="Nicolas Webersinke" title="Product lead" image="/img/nico.jpeg"></Profile>,
    )
    expect(baseElement).toMatchSnapshot()
  })
})
