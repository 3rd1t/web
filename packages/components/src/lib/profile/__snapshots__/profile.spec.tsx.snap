// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile should render successfully 1`] = `
<body>
  <div>
    <div
      class="w-1/4"
    >
      <div
        class="flex flex-col items-center justify-center text-sm "
      >
        <img
          alt="Nicolas Webersinke"
          class="rounded-full"
          src="/img/nico.jpeg"
        />
        <span
          class="mt-4 font-semibold text-gray-900"
        >
          Nicolas Webersinke
        </span>
        <span
          class="text-gray-600"
        >
          Product lead
        </span>
      </div>
    </div>
  </div>
</body>
`;
